packages/stack/test-runner/src/lib/index.js

```diff
@@ -17,6 +17,7 @@ const reports = require('istanbul-reports');
 const Reporter = require('./reporter');
 
 const EMBARK_OPTION = 'embark';
+const GANACHE_CLIENT_VERSION_NAME = "EthereumJS TestRPC";
```

Comment:
This should probably be in the `ganache` plugin package, since it's the package that introduces the ganache simulator provider

Reply:
Hm... Not sure I agree with that. Here's why:

1. It's the test-runner that specifically decides that this version should be treated in a special way (it could be any other version)
2. This would make the stack component dependent on a plugin, which I think is something we don't want to do

Reply:
Yeah, you're right, I commented on that too quick, before seeing what it was used for.
You can disregard this comment.